src/Zap.sol

```diff
@@ -510,7 +505,7 @@ contract Zap is Reentrancy, Errors, Flush(msg.sender) {
         // whatever collateral amount is remaining is returned to the caller
         unwound -= _collateral == USDC
             ? _flashloan
-            : _swapFor(_collateral, _path, _flashloan, _swapMaxAmountIn);
+            : odosSwap(_collateral, _swapAmountIn, _path);
```

Comment:
`odosSwap` returns USDC amount out, when in this case we need the collateral amount spent in the swap (in this case `_swapAmountIn`), otherwise this incorrectly subtracts from unwound balance the USDC amount instead of spent collateral amount